Logger/FileLogger.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string path;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _Path;
```